logproc.go

```diff
@@ -113,18 +113,23 @@ func logProcess(in chan *logData, out chan *logMetrics) {
 		}
 
 		log.Debugln(*data.line)
-		output := strings.Split(*data.line, " - ")
+		var output []string
```

Comment:
Not sure why you change this. `:=` was enough.

Reply:
Indeed, my bad 👍